install_config/storage_examples/gluster_example.adoc

```diff
@@ -83,15 +92,16 @@ Verify that the endpoints were created:
 
 ====
 ----
-# oc get endpoints gluster-endpoints 
+# oc get endpoints gluster-endpoints
 NAME                ENDPOINTS                           AGE
 gluster-endpoints   192.168.122.21:1,192.168.122.22:1   1m
 ----
 ====
 
+[[complete-example-using-gusterfs-creating-the-persistent-volume]]
 == Creating the Persistent Volume
-The persistent volume (PV) file is defined next before creating the PV object in
-OpenShift:
+Next, the persistent volume (PV) file is defined, before creating the PV object
+in {product-title}:
 
```

Comment:
I don't think the comma is needed here. Unless you change the sentence to:

"Next, before creating the PV object, define the persistent volume:"